ci/tasks.py

```diff
@@ -76,6 +80,7 @@ def promote_task(
 
 
 def build_task(
+    use_secrets_server: bool,
```

Comment:
I would prefer to pass the secrets-file here. in general, it was to my experience almost never a good idea to use booleans for this kind of parametrisations